docs/configuration.rst

```diff
@@ -210,6 +210,14 @@ below:
      - 512
      - The maximum length of ``x-datadog-tags`` header allowed in the Datadog propagation style. Must be a value between 0 to 512. If 0, propagation of ``x-datadog-tags`` is disabled.
 
+       .. _dd-trace-x-datadog-tags-propagate-service:
+   * - ``DD_TRACE_PROPAGATE_SERVICE``
+     - Boolean
+     - False
```

Comment:
Should this be True as the default value? I thought the plan was to make this true by default so it's not opt-in and we can reliably give customers improved ingestion data? Then for Govcloud etc, we tell them to set it False.

Reply:
good catch!